src/app/main.js

```diff
@@ -0,0 +1,74 @@
+import app from 'ampersand-app';
+import webChannel from './lib/web_channel';
+import ViewSwitcher from 'ampersand-view-switcher';
+
+import ExperimentsCollection from './collections/experiments';
+import Me from './models/me';
+import HomePage from './views/home_page';
+import Router from './lib/router';
+
+app.extend({
+  initialize() {
+    window.app = this;
+
+    // for now, instantiate the collection with dummy data
+    app.experiments = new ExperimentsCollection([
+      {
+        displayName: 'Universal Search',
+        name: 'universal-search',
+        isInstalled: false
+      }, {
+        displayName: 'Side Tabs',
+        name: 'side-tabs',
+        isInstalled: false
+      }, {
+        displayName: 'Snooze Tabs',
+        name: 'snooze-tabs',
+        isInstalled: false
+      }, {
+        displayName: 'Cheese Tabs',
+        name: 'cheese-tabs',
+        isInstalled: false
+      }
+    ]);
+    // empty (for now) user model
+    // trying out ampersand's 'me' convention, though I find it kinda dumb
+    app.me = new Me();
+
+    // ping the addon to see if it's installed
+    this.listenTo(webChannel, 'from-addon-to-web', (data) => alert(data));
```

Comment:
I would recommend switching this to a `console.log` soon, as the addon emits a _lot_ of progress events.